modules/tflite-micro/CMakeLists.txt

```
@@ -14,6 +15,14 @@ if(CONFIG_TENSORFLOW_LITE_MICRO)
     ${TENSORFLOW_LITE_MICRO_DIR}/third_party_static/ruy
   )
 
+  if (CONFIG_TENSORFLOW_LITE_MICRO_CMSIS_NN_KERNELS)
+    set(CMSIS_NN_OPTIMIZED_KERNEL_DIR cmsis_nn)
+    set(tflm_cmsis_nn_glue_path ${ZEPHYR_CMSIS_MODULE_DIR})
+
+    zephyr_include_directories(${tflm_cmsis_nn_glue_path})
```

assuming this is only required for building the "tflite-micro library," this should use `zephyr_library_include_directories`.

```suggestion
    zephyr_library_include_directories(${tflm_cmsis_nn_glue_path})
```